core/session.rb

```diff
@@ -761,7 +763,13 @@ def up(args)
       end
     end
     $out.info 'All nodes successfully up!'
+    puts "DIR_PWD=#{pwd}"
+    puts "CONF_PATH=#{config_path}"
     Dir.chdir pwd
+    $out.info "Generating #{config_path}_network_settings file"
```

Comment:
Please add some output which will inform user that in case of config/NodeName mode network_setting generation is disabled.